dependencies.props

```diff
@@ -1,15 +1,18 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
   <PropertyGroup>
-    <CoreFxCurrentRef>368eeab3bf02cc55086e170d500cfbb15c974acc</CoreFxCurrentRef>
-    <CoreClrCurrentRef>f4e6d7f7e3c10cbd87d5d9e0fcee32e9aa19f6d1</CoreClrCurrentRef>
+    <CoreFxCurrentRef>534770cbd98c4083de4d93b809f2e0968f0f856a</CoreFxCurrentRef>
     <WCFCurrentRef>c9a03fe0c3309508cb509a4ffdd6dc8b79ed525e</WCFCurrentRef>
+    <StandardCurrentRef>8f8313cbe4b0f4d89c4d0835e08e3e340a18eec0</StandardCurrentRef>
   </PropertyGroup>
 
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
-    <CoreFxExpectedPrerelease>preview2-25309-01</CoreFxExpectedPrerelease>
-    <StandardExpectedPrerelease>preview2-25309-01</StandardExpectedPrerelease>
+    <CoreFxExpectedPrerelease>preview2-25405-01</CoreFxExpectedPrerelease>
+    <StandardExpectedPrerelease>preview2-25401-01</StandardExpectedPrerelease>
```

Comment:
I see, it's being used in a few other places that need to be replaced too:
https://github.com/dotnet/wcf/blob/71d7060ebe62dd590b2c35df44b40b56e6885cbd/src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj#L68
https://github.com/dotnet/wcf/blob/71d7060ebe62dd590b2c35df44b40b56e6885cbd/dir.targets#L119

I did a github search in `master` and changed the branch to `release/2.0.0` to find those two, so there may be more lurking somewhere.

An alternative is to add a `XmlUpdateStep` for `StandardExpectedPrerelease` so it at least stays in sync with `NETStandardPackageVersion`, but replacing is better in the long run. (The 2.0.0 is specifically something that will change with servicing releases, so even in the servicing branch, it's worth fixing.)